src/skmultiflow/meta/leverage_bagging.py

```diff
@@ -244,7 +244,8 @@ def __partial_fit(self, X, y):
                     k = 1.0
                 elif pred[0] != y:
                     k = 1.0
-                elif self._random_state.rand() < (error / (1.0 - error)):
+                elif (error != 1.0 and
+                      self._random_state.rand() < (error / (1.0 - error))):
```

Comment:
Could you elaborate on the reason for the additional check (`error != 1.0`)?

Reply:
This part of the code was unprotected and resulted in a `DivisionByZeroError` when `error==1.`

Reply:
Got it! Thank you for the answer!